taccsite_cms/templates/guides/data_transfer.html

```diff
@@ -1,10 +1,11 @@
 {% extends "guide.html" %}
+{% load static %}
 
 {% block guide %}
   <style>
   /* To override 800px (from stylesheet) which is too big for these images */
   .s-document img {
-    max-width: 570px;
+    max-width: 670px;
```

Comment:
**Explanation.** Image text in Data Transfer's bottom-most images was easier to read if a bit larger.